create_runs.go

```diff
@@ -4,15 +4,16 @@ import (
 	"encoding/csv"
 	"io"
 	"sort"
+	"strings"
 
 	"github.com/pkg/errors"
 )
 
-func (e *ExtSort) createRuns(reader io.Reader) ([]io.ReadSeeker, []func() error, error) {
+func (e *ExtSort) createRuns(reader io.Reader, dst io.Writer) ([]io.ReadSeeker, []func() error, error) {
```

Comment:
Here, the destination is passed to write headers to it. This will not be needed as 
https://github.com/siddhant2408/external-merge-sort/blob/1a61dac8507216afd81ca96b5ce62f069fb26e69/create_runs.go#L59

We are already setting the headers in the main e struct. When writing to the dst reader here,
https://github.com/siddhant2408/external-merge-sort/blob/1a61dac8507216afd81ca96b5ce62f069fb26e69/merge_runs.go#L67

we can read the map and write to dst

Reply:
ok